common/api/bity.ts

```diff
@@ -1,10 +1,34 @@
 import bityConfig, { WhitelistedCoins } from 'config/bity';
 import { checkHttpStatus, parseJSON, filter } from './utils';
+import bitcoinIcon from 'assets/images/bitcoin.png';
+import repIcon from 'assets/images/augur.png';
+import etherIcon from 'assets/images/ether.png';
 
 const isCryptoPair = (from: string, to: string, arr: WhitelistedCoins[]) => {
   return filter(from, arr) && filter(to, arr);
 };
 
+const btcOptions = {
```

Comment:
Below are default fallback icon and name details for Bity swap pairs if Shapeshift fails.